test/e2e/framework/kubernetes/port-forward.go

```diff
@@ -32,6 +32,7 @@ var (
 
 type PortForward struct {
 	Namespace             string
+	PodNamespace          string
```

Comment:
Hmm I would say the dueling namespace usage might be confusing, we should just make the affinity namespace be all namespaces, set to the `metav1.NamespaceAll` const, 

https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#pkg-constants

Reply:
I agree that it's definitely more clearer! Updated PR. Thank you @matmerr